src/authorizations/components/TokenRow.tsx

```diff
@@ -25,8 +25,8 @@ import {Context} from 'src/clockface'
 // Types
 import {Authorization} from 'src/types'
 import {
-  DEFAULT_TOKEN_DESCRIPTION,
-  UPDATED_AT_TIME_FORMAT,
+  DEFAULT_TOKEN_DESCRIPTION, 
+  UPDATED_AT_TIME_FORMAT
```

Comment:
Was this an accidental change or is there something not set correctly in your code editor?

Reply:
yeah, our anal retentive linter is going to complain about this. we'll need to set up your text editor to do this for you.

in the mean time, you can run ```yarn prettier:fix``` locally to fix these